miqsel/miqsel.py

```diff
@@ -175,10 +167,11 @@ def start(ctx, connection):
     container = get_container()
     img = connection.conf.get("image")
     name = connection.conf.get("container_name")
-    
+
     if not container:
-        if not get_image():
-            print("Pulling docker images...Please wait...It will take some time!")
+        if not connection.client.images.list():
```

Comment:
@ganeshhubale this will not work always...
suppose i have `fedora` image and wont have our required docker image then these messages will skip.
You need to check for specific image in `images.list()`

Reply:
@digitronik 
Yes, I am agree with you.
Adding update for checking whether particular image: ```cfmeqe/cfme_sel_stable:latest``` is available or not.

Reply:
- Now I have used ```try..except``` in test because we can get particular exception here as ```docker.errors.ImageNotFound```
- But we can either go with checking image in list: ```if not img in connection.client.images.list():```
Let me know if current code needs any improvement.